app/shared/src/__tests__/GestureController.test.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import { NativeModules } from 'react-native';
+
+describe('GestureController', () => {
+  it('should call NativeModules.RNKiwiGestureController.enableGestures', () => {
+    const GestureController = require('../GestureController').default;
+
+    GestureController.enableGestures('KiwiHotels');
+    expect(
+      NativeModules.RNKiwiGestureController.enableGestures,
+    ).toHaveBeenCalledWith('KiwiHotels');
+  });
+
+  it('should call noop when there is no native module present', () => {
+    jest.resetModules();
+
+    const OriginalGestureController = NativeModules.RNKiwiGestureController;
+
+    /* $FlowFixMe */
```

Comment:
Please use `// $FlowExpectedError:` and explain why you are suppressing the error